src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const listTable = [...document.querySelectorAll('tr')].slice(1, -1);
+const headerTable = document.querySelector('tr');
+const table = document.querySelector('tbody');
+
+const convertNum = salary => +salary.slice(1).split(',').join('');
+
+headerTable.addEventListener('click', e => {
+  let sort;
+
+  switch (e.target.innerText) {
+
+    case 'Name':
+      sort = listTable.sort((a, b) => {
+        return a.children[0].innerText.localeCompare(b.children[0].innerText);
+      });
+
+      table.append(...sort);
+      break;
+
+    case 'Position':
+      sort = listTable.sort((a, b) => {
+        return a.children[1].innerText.localeCompare(b.children[1].innerText);
+      });
+
+      table.append(...sort);
+      break;
```

Comment:
I'd take check the columns better and use a number of colums in children attribute `children[numOfColumn]` to avoid the same check twice (localeCompare).